recipe/meta.yaml

```diff
@@ -51,10 +48,13 @@ about:
   home: https://github.com/kubernetes-incubator/client-python
```

Comment:
There seems to be a redirect in the `Home URL` to the following location:
https://github.com/kubernetes-client/python

Reply:
done